akka-docs/src/main/paradox/remoting-artery.md

```diff
@@ -42,14 +42,14 @@ If migrating from classic remoting see @ref:[what's new in Artery](#what-is-new-
 
 ## Configuration
 
+We recommend using @ref:[Akka Cluster](cluster-usage.md) over using remoting directly. Therefore
+`akka.actor.provider = cluster` should be configured.
+  
 To enable remote capabilities in your Akka project you should, at a minimum, add the following changes
 to your `application.conf` file:
 
 ```
 akka {
-  actor {
-    provider = cluster 
-  }
```

Comment:
I saw Johan's comment, but it will not work without setting the `provider`. Then it will be `local` and no remoting capabilities.

Maybe we could have something like this:
```
akka {
   actor {
     # provider=remote is possible, but prefer cluster
     provider = cluster     
   }
```

Reply:
WFM, updating.